source/MissionAction.h

```diff
@@ -71,18 +80,23 @@ class MissionAction {
 
 	// "Instantiate" this action by filling in the wildcard text for the actual
 	// destination, payment, cargo, etc.
-	MissionAction Instantiate(std::map<std::string, std::string> &subs,
+	MissionAction Instantiate(const ConditionsStore &store, std::map<std::string, std::string> &subs,
 		const System *origin, int jumps, int64_t payload) const;
 
 	int64_t Payment() const noexcept;
 
+
+private:
+	static std::shared_ptr<ConditionSet> GetToDisplay(const DataNode &node);
```

Comment:
This method should either be in the anonymous namespace or a lambda inside the LoadDialog method.
And, it shouldn't involve shared pointers.

Reply:
That method no longer exists.